### PR TITLE
Android sdk version updated

### PR DESCRIPTION
**Pull request by DHRUVA-N-L-CHOUDHARY**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ if (localPropertiesFile.exists()) {
 
 def flutterRoot = localProperties.getProperty('flutter.sdk')
 if (flutterRoot == null) {
-    throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
```
Comment (kamikazechaser) [inline android/app/build.gradle]: I believe `throw new`  is the correct gradle way to do it.

**Review by kamikazechaser — APPROVED** [review]
LGTM

@blayziken Please confirm if this builds on an Android setup

**Comment by blayziken** [comment]
Yh, it does on mine. But I think we should increase the target to 31 because of Android 11 devices

**Comment by kamikazechaser** [comment]
31 had some issues during the build stage. Could you test if t works on yours?

**Pull request merged**